packages/admin-ui/ui/src/components/templates/reservations-table/index.tsx

```diff
@@ -0,0 +1,358 @@
+            {headerGroups?.map((headerGroup) => {
+              const { key, ...rest } = headerGroup.getHeaderGroupProps()
+
+              return (
+                <Table.HeadRow key={key} {...rest}>
+                  {headerGroup.headers.map((col) => {
+                    const { key, ...rest } = col.getHeaderProps()
+                    return (
+                      <Table.HeadCell
+                        className="min-w-[100px]"
+                        key={key}
+                        {...rest}
+                      >
+                        {col.render("Header")}
+                      </Table.HeadCell>
+                    )
+                  })}
+                </Table.HeadRow>
+              )
+            })}
+          </Table.Head>
+
+          <Table.Body {...getTableBodyProps()}>
+            {rows.map((row) => {
+              prepareRow(row)
+              const { key, ...rest } = row.getRowProps()
+              return <ReservationRow row={row} key={key} {...rest} />
+            })}
+          </Table.Body>
+        </Table>
+      </TableContainer>
+      <Fade isVisible={createReservationState} isFullScreen={true}>
+        <NewReservation onClose={closeReservationCreate} />
+      </Fade>
+    </>
+  )
+}
+
+const ReservationRow = ({
+  row,
+  ...rest
+}: {
+  row: Row<ReservationItemDTO>
+} & TableRowProps) => {
+  const inventory = row.original
+
+  const { mutate: deleteReservation } = useAdminDeleteReservation(inventory.id)
+
+  const [showEditReservation, setShowEditReservation] =
+    useState<ReservationItemDTO | null>(null)
+  const [showDeleteReservation, setShowDeleteReservation] = useState(false)
+
+  const getRowActionables = () => {
+    const actions = [
+      {
+        label: "Edit",
+        onClick: () => setShowEditReservation(row.original),
+        icon: <EditIcon size={20} />,
+      },
+      {
+        label: "Delete",
+        variant: "danger",
+        icon: <TrashIcon size={20} />,
+        onClick: () => setShowDeleteReservation(true),
+      },
+    ]
+
+    return actions
+  }
+
+  return (
+    <>
+      <Table.Row
+        color={"inherit"}
+        forceDropdown
+        actions={getRowActionables()}
+        {...rest}
+      >
+        {row.cells.map((cell: Cell, index: number) => {
+          const { key, ...rest } = cell.getCellProps()
+          return (
+            <Table.Cell {...rest} key={key}>
+              {cell.render("Cell", { index })}
+            </Table.Cell>
+          )
+        })}
+      </Table.Row>
+      {showEditReservation && (
+        <EditAllocationDrawer
+          close={() => setShowEditReservation(null)}
+          reservation={row.original}
+        />
+      )}
+      {showDeleteReservation && (
+        <DeletePrompt
+          text={"Are you sure you want to remove this reservation?"}
+          heading={"Remove reservation"}
+          successText={"Reservation has been removed"}
+          onDelete={deleteReservation}
```

Comment:
todo: The notification on deleting reservations has disappeared

Reply:
I fixed the missing notification, I think the issue was related to passing the mutation method, but looks good now =